src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -1754,23 +1754,25 @@ def _get_enable_managed_identity(
         """
         # read the original value passed by the command
         enable_managed_identity = self.raw_param.get("enable_managed_identity")
-        # try to read the property value corresponding to the parameter from the `mc` object
+        # In create mode, try to read the property value corresponding to the parameter from the `mc` object
```

Comment:
Currently we are refactoring `aks create` and `aks update`, and "In create mode" means this only applies to the case of `aks create` but not `aks update`.